docs/server-instructions.md

```diff
@@ -0,0 +1,106 @@
+# Add the JVM agent to Java servers
+
+The following sections show how to add the path to the Splunk OTel agent for Java using the [supported servers](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#application-servers).
+
+## JBoss EAP / WildFly
+
+Add the `javaagent` argument to the `standalone` configuration file:
+
+- On Linux and macOS, add the following line at the end of the `standalone.conf` file:
+   ```
+   JAVA_OPTS="$JAVA_OPTS -javaagent:/path/to/splunk-otel-javaagent.jar"
+   ```
+- On Windows, dd the following line at the end of the `standalone.conf.bat` file:
+   ```
+   set "JAVA_OPTS=%JAVA_OPTS% -javaagent:<Drive>:\path\to\splunk-otel-javaagent.jar"
+   ```
+
+## Jetty
+
+Add the path to the JVM agent using the `-javaagent` argument:
+
+```
+java -javaagent:/path/to/splunk-otel-javaagent.jar -jar start.jar
+```
+
+Alternatively, depending on how you define JVM arguments, you can edit your `jetty.sh` or `start.ini` files, :
+
+-  If you use the `jetty.sh` file, add the `javaagent` argument:
+   ```
+   JAVA_OPTIONS="${JAVA_OPTIONS} -javaagent:/path/to/splunk-otel-javaagent.jar"
+   ```
+- If you use the `start.ini` file, add the `javaagent` argument right below the `--exec` option:
+   ```
+   #===========================================================
+   # Sample Jetty start.ini file
+   #-----------------------------------------------------------
+   --exec
+   -javaagent:/path/to/splunk-otel-javaagent.jar
+   ```
+
+## Glassfish / Payara
+
+Add the path to the JVM agent to the settings using the `asadmin` command-line tool:
+
+- On Linux, enter the following command:
+   ```
+   <server_install_dir>/bin/asadmin create-jvm-options "-javaagent\:/path/to/splunk-otel-javaagent.jar" 
+   ```
+- On Windows, enter the following command:
+   ```
+   <server_install_dir>\bin\asadmin.bat create-jvm-options '-javaagent:<Drive>:\path\to\splunk-otel-javaagent.jar'
```

Comment:
Would it be possible to verify this on windows? Based on the example in https://docs.oracle.com/cd/E26576_01/doc.312/e24938/create-jvm-options.htm#GSRFM00042 I suspect that it should really be `'-javaagent\:<Drive>\:\\path\\to\\splunk-otel-javaagent.jar'` Perhaps there should also be a note that `:` and `\` need to be escaped

Reply:
From https://stackoverflow.com/a/28040996

> The asadmin manual page says the following:
> 
> For the Windows operating system in single mode, a backslash is required to escape the colon and the backslash characters.
> 
> So try the following:
> 
> -javaagent:C\:\\AppServerAgent\\javaagent.jar